layers/meta-resin-iot2000/conf/samples/bblayers.conf.sample

```diff
@@ -0,0 +1,28 @@
+# LAYER_CONF_VERSION is increased each time build/conf/bblayers.conf
+# changes incompatibly
+LCONF_VERSION = "6"
```

Comment:
The above should be replaced by:

POKY_BBLAYERS_CONF_VERSION = "2"

Reply:
Please use an autogenerated file from a build.